src/pages/components/image-with-caption.mdx

```diff
@@ -29,6 +30,16 @@ The Image with caption supports images at 1x1, 2x1, 4x3, and 16x9 aspect ratios.
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline (caption) | Short copy | Yes | 1 | XXS (40 / 55)|
+| Description | Long copy | No | 1 | L (150 / 200)|
```

Comment:
So I love the short and long field types as a descriptor, but just as a reminder (I'm sure you know because I say it every chance I get) we don't set character limits on our components.

Reply:
Is it important to call out someway how these are being used? Like description is used for the light box media viewer?

Reply:
Haha yep, i do remember that :) That's a good point. I mentioned it slightly in light box media viewer, but I can add more context to better help the user.